.github/sla.yml

```diff
@@ -6,6 +6,7 @@
         - WaitForARMFeedback
         - DoNotMerge
         - WaitingForAzureAPIBoard
+        - ARMSignedOff
```

Comment:
add one more `AzureAPIBoardSignedOff`

Reply:
> add one more AzureAPIBoardSignedOff

done

Reply:
second thought on filtering `ARMSignedOff` label, it's ok to filter out those labels in private repo, but not in public repo. After a PR is labelled with "ARMSignedOff`, if no `DoNotMerge` label at same time, the PR owner should be notified to merge the PR as soon as possible. Can we do combination filtering? such as 
if `DoNotMerge` and 'ARMSignedOff`, then no ping;
If 'DoNotMerge` and !`ARMSignedOff`, then ping;

Reply:
For complex conditions. I think I can develop a new feature about evaluating label condition expressions. For example: condition: `(DoNotMerge&&!ARMSignedOff)||WaitForARMFeedback`. 

For public and private repo syncing issues. I think this is a special case. I can predefine repository variable and add it in condition sections or other defined sections. For example:  `$REPOSITORY_NAME==Azure/azure-rest-api-specs`

It's a more flexible solution.

Reply:
sounds good. Then what's the plan for this PR?

Reply:
I suggest we can merge it to master. because it will bother rehe every day during the holiday.